### PR TITLE
52 Weeks of capa Rules - 2

### PR DESCRIPTION
**Pull request by mike-hunhoff**
adding/updating rules for `.NET`.

### CODE REVIEW

File: communication/tcp/serve/start-tcp-server.yml [hunk 1]
```diff
@@ -4,15 +4,24 @@ rule:
...
+        - or:
+          - api: accept
+          - api: WSAAccept
+      - api: System.Net.Sockets.TcpListener::Start
+      - api: System.Net.Sockets.TcpListener::AcceptTcpClient
+      - api: System.Net.Sockets.TcpListener::BeginAcceptTcpClient
+      - api: System.Net.Sockets.TcpListener::AcceptTcpClientAsync
+      - api: System.Net.Sockets.TcpListener::AcceptSocket
+      - api: System.Net.Sockets.TcpListener::BeginAcceptSocket
+      - api: System.Net.Sockets.TcpListener::AcceptSocketAsync
```
Comment (mr-tz) [inline communication/tcp/serve/start-tcp-server.yml:27]:
```suggestion
      - api: System.Net.Sockets.TcpListener::AcceptSocketAsync

```

File: nursery/decode-data-using-url-encoding.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: decode data using URL encoding
```
Comment (mr-tz) [inline nursery/decode-data-using-url-encoding.yml:3]: `decode URL-encoded data`?

File: nursery/manipulate-user-privileges.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: manipulate user privileges
```
Comment (mr-tz) [inline nursery/manipulate-user-privileges.yml:3]: add this to `modify access privileges`?

File: nursery/decode-data-using-url-encoding.yml (hunk 2 above)
Comment (mike-hunhoff) [reply nursery/decode-data-using-url-encoding.yml:3]: This was my initial choice but it does not adhere to the naming scheme we have been using e.g.
- `decode data using Base64 in .NET`
- `decrypt data using RSA`
- `decrypt data via SSPI`

Happy to make the suggested change pending additional thoughts

File: nursery/manipulate-user-privileges.yml (hunk 3 above)
Comment (mike-hunhoff) [reply nursery/manipulate-user-privileges.yml:3]: `modify access privileges` appears specific to a process whereas `LsaAddAccountRights` is specific to a user which is why I kept them separate.  Thoughts?

File: nursery/decode-data-using-url-encoding.yml (hunk 2 above)
Comment (mr-tz) [reply nursery/decode-data-using-url-encoding.yml:3]: Hm, right, I'm fine leaving as is then.

File: nursery/manipulate-user-privileges.yml (hunk 3 above)
Comment (mr-tz) [reply nursery/manipulate-user-privileges.yml:3]: ah, right, never mind

File: nursery/manipulate-user-privileges.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    namespace: communication/authentication
```
Comment (mr-tz) [inline nursery/manipulate-user-privileges.yml:4]:
```suggestion
    namespace: host-interaction/user
```

**Pull request merged**